subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -175,13 +175,32 @@ If a project defines multiple publications then Gradle will publish each of thes
 Publications are published to repositories. The repositories to publish to are defined by the api:org.gradle.api.publish.PublishingExtension#getRepositories()[] container.
 
 ++++
-<sample dir="maven-publish/quickstart" id="publishing_maven:repositories" title="Declaring repositories to publish to">
+<sample dir="maven-publish/quickstart" id="publishing_maven:example:repositories" title="Declaring repositories to publish to">
     <sourcefile file="build.gradle" snippet="repositories"/>
 </sample>
 ++++
 
 The DSL used to declare repositories for publication is the same DSL that is used to declare repositories to consume dependencies from, api:org.gradle.api.artifacts.dsl.RepositoryHandler[]. However, in the context of Maven publication only api:org.gradle.api.artifacts.repositories.MavenArtifactRepository[] repositories can be used for publication.
 
+[[publishing_maven:snapshot_and_release_repositories]]
+==== Snapshot and release repositories
+
+It is a common practice to publish snapshots and releases to different Maven repositories. A simple way to accomplish this is to configure the repository URL based on the project version.
```

Comment:
When you're leading into an example, end the paragraph with a colon. Alternatively, explicitly mention what you're demonstrating in the sample that follows the paragraph.

For example:

    A simple way to accomplish this is to configure the repository URL based on the project version. The following sample uses one URL for versions that end with "-SNAPSHOT" and a different URL for the rest:

BTW, I got lost amongst all the START/END SNIPPET lines. I assume interleaving those works. Is that correct?